providers/mongodb.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-shadow */
+let db;
+const config = {
+  moduleName: "mongo",
+  enabled: true,
+  dbName: "Komada",
+  dbURL: "mongodb://localhost:27017/",
+};
+
+function connect() { db.connect(`${config.dbURL}${config.dbName}`); }
+
+// Collection Methods. Collections are implicitly created with document methods regardess.
+
+/**
+ * Create a collection within a DB. Options may be specfied, refer to MongoDB docs.
+ *
+ * @param {string} name Name of the Collection to crate
+ * @param {Object} [options] Object containing various options for the created Collection
+ * @returns {Promise} Returns a promise containing the created Collection.
+ */
+exports.createCollection = (name, options = {}) => connect().then(db => db.createCollection(name, options));
+
+/**
+ * Drops a collection within a DB.
+ *
+ * @param {string} name
```

Comment:
You must write a description for the parameter `name`. Same for the rest of JSDocs.